src/rgw/rgw_redis_driver.cc

```diff
@@ -523,7 +523,7 @@ int RedisDriver::set_attr(const DoutPrefixProvider* dpp, const std::string& key,
   return std::get<0>(resp).value();
 }
 
-static Aio::OpFunc redis_read_op(optional_yield y, std::shared_ptr<connection> conn,
+Aio::OpFunc RedisDriver::redis_read_op(optional_yield y, std::shared_ptr<connection> conn,
```

Comment:
Isn't this method used only by this source file (and no other translation unit), so it makes sense to let this be static.

Reply:
It is statically defined in the header file so I believe it should still be `static`